ivy/functional/backends/tensorflow/__init__.py

```diff
@@ -148,6 +150,45 @@
 
 native_inplace_support = False
 
+backend = "tensorflow"
+
+
+def _handle_inplace_mode():
+    if not hasattr(ivy, "inplace_mode") or ivy.inplace_mode not in [
+        "lenient",
+        "strict",
+    ]:
+        ivy.set_inplace_mode("lenient")
```

Comment:
I think the `inplace_mode_stack` should be empty by default, we can keep `"lenient"` as the default mode as explained in the comments below